tslint.json

```diff
@@ -113,6 +113,7 @@
         "jsx-no-multiline-js": false,
         "jsx-wrap-multiline": false,
         "no-var-requires": false,
-        "no-empty": false
+        "no-empty": false,
+        "member-ordering": false
```

Comment:
[member-ordering](https://palantir.github.io/tslint/rules/member-ordering/): this rule is about define all public methods before private methods, not about declare methods before use it.

What do you think about disable this rule?